scalafix-tests/output/src/main/scala/test/explicitResultTypes/ExplicitResultTypesInfix.scala

```diff
@@ -0,0 +1,12 @@
+package test.explicitResultTypes
+
+import test.explicitResultTypes.{ ExplicitResultTypesInfix, \/ }
+trait \/[A, B]
+
+object ExplicitResultTypesInfix {
+  val ab: Int \/ String = null.asInstanceOf[Int \/ String]
+  def foo: Int \/ String = ab
+
+  case class :+:[T1, T2]()
+  def bar: ExplicitResultTypesInfix.:+:[Double, Long] = :+:[Double, Long]()
```

Comment:
btw is this the best we can do for now? this type has a prefix, but it's unnecessary in this context, heh

Reply:
@olafurpg